slices/gsettings-desktop-schemas.yaml

```
@@ -0,0 +1,7 @@
+package: gsettings-desktop-schemas
+
+slices:
+  data:
+    contents:
+      /usr/share/GConf/gsettings/*:
+      /usr/share/glib-2.0/schemas/*:
```

```suggestion
      /usr/share/glib-2.0/schemas/10_gsettings-desktop-schemas.gschema.override:
      /usr/share/glib-2.0/schemas/org.gnome.desktop.*.gschema.xml:
      /usr/share/glib-2.0/schemas/org.gnome.desktop.enums.xml:
      /usr/share/glib-2.0/schemas/org.gnome.system.*.gschema.xml:      
```

(I think this should work, but please double check :) )

So, like in previous comments, I prefer to encourage a conservative use of globs. Although they can make slices much smaller, at scale, they can also become a burden and hard to trace if they are too generic. Whenever possible, I'd try to find a sweet spot between simplicity, maintainability and explicitness. In this example, I've sacrificed 3 lines of the slice, but not only it makes the slice easier to read, but it also saves us from potential collisions with other packages in the future.